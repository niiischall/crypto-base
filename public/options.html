<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="theme-color" content="#000000" />
    <meta
      name="description"
      content="extension created using create-react-extension"
    />
    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
    <title>React App</title>
  </head>
  <body>
    <noscript>You need to enable JavaScript to run this app.</noscript>
    <div id="options"></div>
  </body>
</html>