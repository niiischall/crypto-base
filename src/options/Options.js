import './Options.css';

export const Options = () => {
  return (
    <div className="App">
      <span>Options!</span>
    </div>
  );
};

export default Options;
