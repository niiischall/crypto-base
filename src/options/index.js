import React from 'react';
import ReactDOM from 'react-dom';

ReactDOM.render(
  <React.StrictMode>
    <span>&nbsp;</span>
  </React.StrictMode>,
  document.getElementById('options'),
);
