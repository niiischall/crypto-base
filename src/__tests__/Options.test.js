import ReactDOM from 'react-dom';

import { Options } from '../options/Options';

describe('Options Page', () => {
  const div = document.createElement('div');

  it('should match snapshot', () => {
    const component = ReactDOM.render(<Options />, div);
    expect(component).toMatchSnapshot();
  });
});
