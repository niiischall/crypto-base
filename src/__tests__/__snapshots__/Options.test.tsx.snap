// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Options Page should match snapshot 1`] = `null`;
